code/game/objects/items/stacks/seaweed.dm

```diff
@@ -0,0 +1,13 @@
+/obj/item/stack/seaweed
```

Comment:
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/b3d39300-baeb-4075-a62f-e39263865016)

Could make the attack_self() return here (attack_self_tk too i think), since theres no use to that interface with seaweeds.